data_selection/static/data_selection.js

```diff
@@ -0,0 +1,255 @@
+  wavesurfer.stop();
+});
+
+// Set the audio speed rate
+$(".rate-option").on("click", function() {
+  let index = wavesurfer.getPlaybackRate();
+  const rates = [0.8, 1, 1.2];
+  if (this.id === "plus") {
+    wavesurfer.setPlaybackRate(rates[index + 1]);
+  } else if (this.id === "minus") {
+    wavesurfer.setPlaybackRate(rates[index - 1]);
+  }
+  let currentRate = wavesurfer.getPlaybackRate();
+  $("#audio-speed-rate").text(currentRate + "x");
+});
+
+// Navigation for the samples
+$(".sample-label span").on("click", function() {
+  let id = this.id;
+  loadAudioSample(this.id);
+  $(".sample-label span").removeClass("active");
+  $("#" + id).addClass("active");
+
+  // Hide the result panel when the user click on another audio sample
+  $("#results-section").addClass("display-none");
+});
+
+// Restart the audio sample (return to the start)
+$("#go-start").on("click", function() {
+  wavesurfer.stop();
+  $("#stop-btn").addClass("display-none");
+  $("#play-btn").removeClass("display-none");
+});
+
+// Finish the audio sample
+$("#go-end").on("click", function() {
+  wavesurfer.skipForward(10);
+  $("#stop-btn").addClass("display-none");
+  $("#play-btn").removeClass("display-none");
+});
+
+let input = "";
+
+$("#input-transcription")
+  .keypress(function(e) {
+    if (e.which === 13) {
+      isLoading();
+      submit(input);
+      clearResults();
+      $("#results-section").addClass("display-none");
+      return false;
+    }
+  })
+  .on("input", function() {
+    input = this.value;
+    checkCompletion();
+  });
+
+$("#submit-button").on("click", function() {
+  isLoading();
+  submit(input);
+  clearResults();
+  $("#results-section").addClass("display-none");
+});
+
+function submit(input) {
+  let currentSampleId = $(".active").attr("id");
+  var url =
+    "http://52.30.254.33:6121/api/compute/sim/" + uttId[currentSampleId];
+  var data = { input: input };
+
+  $.ajax({
+    type: "POST",
+    url: url,
+    contentType: "application/json",
+    data: JSON.stringify(data),
+    success: function(data) {
+      let jsonPhonSim = data["ng"];
+      let jsonLexiSim = data["tp"];
+
+      let phoneticalSimilarities = buildResultStructure(jsonPhonSim);
+      let lexicalSimilarities = buildResultStructure(jsonLexiSim);
+
+      $("#user-transcription").append(
+        '<div class="user-transcription">User transcription:</div>' + input
+      );
+      $("#input-transcription")
+        .prop("disabled", false)
+        .val("");
+
+      $("#loader").addClass("display-none");
+      $("#submit-button span").show();
+
+      input = "";
+      checkCompletion();
+
+      $("#phon-root").append(phoneticalSimilarities);
+      $("#lex-root").append(lexicalSimilarities);
+      $("#results-section").removeClass("display-none");
+    }
+  });
+}
+
+$(document).ready(function() {
+  $("#audio-speed-rate").text(wavesurfer.audioRate);
+  loadAudioSample(0);
+});
+
+// Functions definition
+const minInputLength = 9;
```

Comment:
I don't know if it's the case in js, but in other languages, const are all upper case